manual_flagging/yaml/inputs_update.yaml

```diff
@@ -2,19 +2,12 @@
 # We use these inputs when we want to re-flag already flagged sales.
 ---
 stat_groups:
-  single_family:
+  condos:
     - "rolling_window"
     - "geography_split"
-    - "bldg_age_bin"
-    - "char_bldg_sf_bin"
-  multi_family:
-    - "rolling_window"
-    - "geography_split"
-    - "bldg_age_bin"
 
 iso_forest: [
   "meta_sale_price",
-  "sv_price_per_sqft",
```

Comment:
**[Question, non-blocking]** Hmm, is this an indication that the `iso_forest` parameter will need to be different between res and condo groups too? Have we factored that into our redesign (#89)?

Reply:
Yes, I would have had to factor this out. Although we didn't scope it during our meeting. Good catch, I'll incorporate that into the config file.